package.json

```diff
@@ -1,21 +1,21 @@
 {
   "name": "diory-browser-electron",
-  "version": "0.3.3",
+  "version": "0.3.4",
   "productName": "Diory - Digital Memory Browser",
   "main": "electron/main.js",
   "private": true,
   "dependencies": {
     "evergreen-ui": "^4.24.0",
-    "exifreader": "^3.6.0",
+    "exifreader": "^2.13.0",
```

Comment:
Can you describe why you downgraded and document it somewhere so we remember 
1) not try to upgrade this package next time (and get the same error)
2) why we shouldn't upgrade this package
3) what we should do before we could upgrade this package

Or is it too much? Or do you have other suggestions?

Reply:
I added these to debt.

Reply:
apparently except exifreader, which I added now